### PR TITLE
Roxygen2 7.0.0

### PR DESCRIPTION
**Pull request by hadley**
TODO:

* [x] Update date

### CODE REVIEW

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+* `@family` automatically adds `()` when linking to functions,
+  and prints each link on its own line (to improve diffs).
+  
+We've also removed a few features to simplify the code and/or clearly advertise that certain features are no longer supported:
+
+* `@S3method` has been removed. It was deprecated in roxygen2 4.0.0
+  released 2014-05-02, over 5 years ago.
+
+* Using the `wrap` option will now trigger a warning, as it hasn't worked
+  for quite some time. Suppress the wraning by remove the option from your
```
Comment (gaborcsardi) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: wraning typo

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  \subsection{Subheading}{
+    Second sentence
+  }
+}
+\section{New section}{
+  Third sentence
+}
+```
+
+Markdown tables used the [GFM table syntax](https://github.github.com/gfm/#tables-extension-) and are converted to `\tabular{}` macros. For example,
```
Comment (gaborcsardi) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: used -> use

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    ```{r child = "vignettes/common.Rmd"}`r ''`
+    ```
+    ````
+
+Learn more in 
+[`vignette("rd")`](https://roxygen2.r-lib.org/articles/rd.html#including-external--rmd-md-files)
+
+## R6 documentation
+
+You can now document R6 classes! The basic usage is straighforward and works similarly to documenting function. The main difference is that methods require explicit `@description` and `@detail` tags.
```
Comment (gaborcsardi) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: "documenting functions"

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    greet = function() {
+      cat(paste0("Hello, my name is ", self$name, ".\n"))
+    }
+  )
+)
+```
+
+Learn more in [`vignette("rd")`](https://roxygen2.r-lib.org/articles/rd.html#r6).
+
+R6 documentation is a work in progress, so please let us know if you find anything missing or confusing. If you document a package with many R6 classes, you will get many warnings about missing documentation. If you want to suppress those warnings, you can turn off R6 documetation with the R6 option, i.e. put `Roxygen: list(r6 = FALSE)` in your `DESCRIPTION`.
```
Comment (gaborcsardi) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: "with the `r6` option"?

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+* `load_source()` attaches required packages and `source()`s all files in `R/`. 
+  This is a cruder simulation of package loading than pkgload (and e.g. is 
+  unreliable if you use S4 extensively), but it does not require that the 
+  package be compiled. Try it if the default strategy (used in roxygen2 6.1.0 
+  and above) causes you grief.
+
+* `load_installed()` assumes you have installed the package. This is best
+  used as part of a bigger automated workflow.
+
+You can override the default either by calling (e.g.) `roxygenise(load_code = "source"))` or by setting the `load` option in your DESCRIPTION: `Roxygen: list(load = "source")`.
```
Comment (gaborcsardi) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: Extra paren in `..."source"))`

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+author: Hadley Wickham
+date: '2019-11-05'
+slug: roxygen2-7-0-0
+categories:
+  - package
+tags:
+  - r-lib
+  - devtools
+  - roxygen2
+photo:
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: Add description in YAML?

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+
+Install the latest version of roxygen2 with:
+
+```{r, eval = FALSE}
+install.packages("roxygen2")
+```
+
+This a huge release containing [many minor improvements and bug fixes](https://roxygen2.r-lib.org/news/index.html#roxygen2-7-0-0). This blog post focusses on seven major improvements:
+
+* roxygen2 is no longer (ironically!) the worst documented package. It has
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
* roxygen2 is no longer (ironically!) the worst documented package. It has a 
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+This a huge release containing [many minor improvements and bug fixes](https://roxygen2.r-lib.org/news/index.html#roxygen2-7-0-0). This blog post focusses on seven major improvements:
+
+* roxygen2 is no longer (ironically!) the worst documented package. It has 
+  fresh new website, <https://roxygen2.r-lib.org>, and the vignettes have 
+  all been updated.
+
+* We've made a number of tweaks to the rendering of `.Rd`. Most importantly
+  you no longer needed to escape `%` in markdown, and functions with many
+  arguments are displayed with one argument per line. We've also removed a few 
+  old features that are no longer supported.
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: If there's a convenient place where you could link to this in the Change Log, that'd be 🆒 .

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+
+* You can create tables and headings from markdown.
+
+* You can share text and code between your README/vignettes, and your 
+  documentation with the new `@includeRmd` tag.
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: If you're saying between your (1) README/vignettes and (2) documentation, then you should axe the comma.

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+    `DESCRIPTION`:
+    
+    ```
+    Roxygen: list(old_usage = TRUE)
+    ```
+
+You'll also notice a number of small improvements:
+
+* Markdown code is converted to to either `\code{}` or `\verb{}`, depending on 
+  whether it not is R code. For example, `` `foofy()` `` will become
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
  whether or not it is R code. For example, `` `foofy()` `` will become 
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown [hunk 11]
```diff
@@ -0,0 +1,281 @@
...
+    ```
+    Roxygen: list(old_usage = TRUE)
+    ```
+
+You'll also notice a number of small improvements:
+
+* Markdown code is converted to to either `\code{}` or `\verb{}`, depending on 
+  whether it not is R code. For example, `` `foofy()` `` will become 
+  `\code{foofy()}` but `` `1 +` `` will become `\verb{1 + }`. This better 
+  matches the intended usage of the `\code{}` and `\verb{}` macros and should
```
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
  matches the intended usage of the `\code{}` and `\verb{}` macros, and should 
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 1 above)
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
  for quite some time. Suppress the warning by remove the option from your
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 2 above)
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
Markdown tables use the [GFM table syntax](https://github.github.com/gfm/#tables-extension-) and are converted to `\tabular{}` macros. For example,
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 3 above)
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
You can now document R6 classes! The basic usage is straighforward and works similarly to documenting functions. The main difference is that methods require explicit `@description` and `@detail` tags.
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 5 above)
Comment (batpigandme) [inline content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]:
```suggestion
You can override the default either by calling (e.g.) `roxygenise(load_code = "source")` or by setting the `load` option in your DESCRIPTION: `Roxygen: list(load = "source")`.
```

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 3 above)
Comment (hadley) [reply content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: You can make suggestions directly (like Mara did) for typos

File: content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown (hunk 8 above)
Comment (hadley) [reply content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown]: Not really, and I discuss them below

**Pull request merged**